JpegXmpWritePluginMDE/JpegXmpWritePluginMDE.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations"/>
+    <PackageReference Include="MetadataExtractor"/>
+    <PackageReference Include="xunit.assert"/>
```

Comment:
Should there be an xunit package attached to the main library project?

Reply:
also (could be done as a separate / followup thing) - all the JetBrains annotations stuff has been removed from the core MDE lib in favour of the built in annoitations - https://github.com/drewnoakes/metadata-extractor-dotnet/pull/387

Reply:
> Should there be an xunit package attached to the main library project

Good catch, cleaned up

Reply:
> also (could be done as a separate / followup thing) - all the JetBrains annotations stuff has been removed from the core MDE lib in favour of the built in annoitations - [drewnoakes/metadata-extractor-dotnet#387](https://github.com/drewnoakes/metadata-extractor-dotnet/pull/387)

I can add this as a follow up task. Created a label to tag a separate PR with: https://github.com/ahsanaman92/JpegXmpWritePluginMDE/labels/RemoveJetbrainAnnotations